### DISCUSSION TITLE
Apple Chip Mac Support

### DISCUSSION THREAD

**Discussion by mberbero** [getezy/ezy#19] [Q&A] [answered]
Will you support Apple Chip Mac?

**Comment by notmedia** [comment, marked as answer]
> Will you support Apple Chip Mac?

Hi @mberbero! Thank you for the first opened question 😉

`ezy` already support Apple chips. Right now the latest version is [ezy-darwin-arm64-1.0.0-beta.8.zip](https://github.com/getezy/ezy/releases/download/v1.0.0-beta.8/ezy-darwin-arm64-1.0.0-beta.8.zip)

**Reply by mberbero** [reply]
<img width="254" alt="Screen Shot 2022-08-25 at 13 33 38" src="https://user-images.githubusercontent.com/40326463/186642729-8c65c4ba-e194-4a97-a6fa-daea05bc4ce5.png">

It doesnt work @notmedia

**Reply by notmedia** [reply]
I don't have m1 right now. Could you check it for me?

In terminal - `xattr -d com.apple.quarantine /path/to/ezy.app`  this will move trusted file out of quarantine. Then try to run app again.

**Reply by mberbero** [reply]
that worked
thanks

**Reply by notmedia** [reply]
Great, the problem here is that I skipped code signing for app distr. I will add this soon if I find a way to pay for Apple Developer Program, you can track it here - https://github.com/getezy/ezy/issues/20

**Reply by notmedia** [reply]
Hey, @mberbero! I'm happy to let you know that all new Apple distributions are signed and notarized.
